From 8bcba014869a5d18caa63b8953dc4ae979957efe Mon Sep 17 00:00:00 2001
From: Austin Delgado <austindelgado@google.com>
Date: Wed, 21 May 2025 12:03:10 -0700
Subject: [PATCH] Revert "Fix biometric prompt appearing above shade"

This reverts commit 93b417adaecdd26c2e99d7cdc5eacac84aa842e3.

Reason for revert: Reverting on old branches

Bug: 384727394
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:1b19eeac213c8b93c2c2a745c666b683da62ccc0)
Merged-In: I0fb69ebb12809cc118c7cc716d670eacc007c5e7
Change-Id: I0fb69ebb12809cc118c7cc716d670eacc007c5e7
---
 .../src/com/android/systemui/biometrics/AuthContainerView.java  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/SystemUI/src/com/android/systemui/biometrics/AuthContainerView.java b/packages/SystemUI/src/com/android/systemui/biometrics/AuthContainerView.java
index b6537118324e..6e85d9e8bbc0 100644
--- a/packages/SystemUI/src/com/android/systemui/biometrics/AuthContainerView.java
+++ b/packages/SystemUI/src/com/android/systemui/biometrics/AuthContainerView.java
@@ -762,7 +762,7 @@ public class AuthContainerView extends LinearLayout
         final WindowManager.LayoutParams lp = new WindowManager.LayoutParams(
                 ViewGroup.LayoutParams.MATCH_PARENT,
                 ViewGroup.LayoutParams.MATCH_PARENT,
-                WindowManager.LayoutParams.TYPE_APPLICATION_OVERLAY,
+                WindowManager.LayoutParams.TYPE_KEYGUARD_DIALOG,
                 windowFlags,
                 PixelFormat.TRANSLUCENT);
         lp.privateFlags |= WindowManager.LayoutParams.SYSTEM_FLAG_SHOW_FOR_ALL_USERS;
-- 
2.34.1

